### ISSUE TITLE
RVPN not implemented

### ISSUE THREAD

**Issue by louptheron** [Daplie/holepunch#3]
Hi,

Could you explain me what RVPN mean ?

`[HP] RVPN not implemented
Error: no port map success
    at /home/distohm/workspace/electron-test/electron-boilerplate/build/node_modules/holepunch/lib/index.js:206:36
`

This error mean my router does not support UPnP and PMP right ?

Thank you

**Comment by coolaj86** [comment]
Yes, the error means that your router doesn't support or doesn't *correctly implement* UPnP / SSDP / NAT-PMP.

We're building an RVPN service so that even when you're at the library or on a locked-down bank network you can still get a public IP address.

The technical demo has been working since last year, but we haven't productized it. I'm actually picking that back up that this / next week.
